addons/account/static/src/components/account_merge_wizard_line_one2many/account_merge_wizard_line_one2many.js

```diff
@@ -0,0 +1,68 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import {
+    SectionAndNoteListRenderer,
+    SectionAndNoteFieldOne2Many,
+    sectionAndNoteFieldOne2Many,
+} from "../section_and_note_fields_backend/section_and_note_fields_backend";
+import { onWillStart } from "@odoo/owl";
+
+export class AccountMergeWizardLinesRenderer extends SectionAndNoteListRenderer {
+    setup() {
+        super.setup();
+        this.titleField = "info";
+    }
+
+    getCellClass(column, record) {
+        const classNames = super.getCellClass(column, record);
+        // Even though the `is_selected` field is hidden for section lines, we should
+        // keep its column to align the section header name with the account names.
+        if (this.isSectionOrNote(record) && column.name === "is_selected") {
+            return classNames.replace(" o_hidden", "");
```

Comment:
Maybe reduce the width of the column? (not sure if it should be done here or in the view)